hearth/spec/hearth/telemetry/span_status_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Hearth
+  module Telemetry
+    describe SpanStatus do
+      describe '.unset' do
+        it 'returns the correct expected code' do
+          status = Hearth::Telemetry::SpanStatus.unset
```

Comment:
This is what my previous comment is referring to about constants. The API expects these methods, so constants shouldn't exist I think.

Reply:
I think the actual expectation should be more like, `expect(status.code).to eq(Hearth::Telemetry::SpanStatus::UNSET)` when re-looking at this

Reply:
Will customers use Hearth::Telemetry::SpanStatus::UNSET or SpanStatus.unset? If the latter, then former constant should be removed and the test was fine as is.

Reply:
Potentially, both could be used. Like below:
Customer could set the span status code like so:
```ruby
span = tracer.start_span('foo')
span.status = Hearth::Telemetry::SpanStatus.ok(description: 'hello!')
```

Customer may want to verify its status by using constant.
```ruby
# maybe some work relating to collecting certain spans based on status
error_spans = []
spans.each do | span |
  if span.status.code == Hearth::Telemetry::SpanStatus::ERROR
  exceptions << span
end
```